CONTRIBUTING.md

```diff
@@ -19,6 +19,23 @@ If you want to make a bigger change, it's a good idea to first file an issue and
 - Make your changes, commit to git, and then create a PR by running `usethis::pr_push()`, and following the prompts in your browser. The title of your PR should briefly describe the change. The body of your PR should contain Fixes #issue-number.
 - For user-facing changes, add a bullet to the top of `NEWS.md` (i.e. just below the first header). Follow the style described in https://style.tidyverse.org/news.html.
 
+### What happens after submitting a pull request?
+
+PRs are reviewed by the team before they are merged. The review process begins after continuous integration checks, which can take some time. We do not expect contributors to familiarise themselves with all the automated checks in our GitHub actions, and are happy to support on this once a PR is made. 
+
+However, for those who are interested in running local checks to accelerate the PR process, here is a summary of the main checks that are automatically performed and how they can be confirmed:
+
+- Load the package for Epiverse-customised versions of the checks: `remotes::install_github("epiverse-trace/etdev")`.
+
+- Run `testthat::test_package()` to check that package tests are still passing.
+
+- Run `lintr::lint_package()` to check for programmatic and stylistic errors in code.
+
+- Run `spelling::spell_check_test()` to check spelling and update word list if necessary.
+
+- Run `devtools::check()` to check overall package structure.
```

Comment:
This does the spell checking and test coverage check so maybe we don't have to list them explicitly? Maybe you can record to say that `devtools::check()` does a range of checks but if the user wants control over it, they can run the individual checks before running `devtools::check()`.

One additional thing to run is `devtools::document()`. If changes were made to documentation, that function needs to be run to regenerate the `.Rd` files.

Reply:
Thanks, good suggestions.